data/human/culture conversations.txt

```diff
@@ -1627,3 +1627,270 @@ mission "A Sunracer Hovercraft Racing"
 				clear "second place margin"
 				clear "last place margin"
 				clear "hovercraft racing: talked to fan"
+
+
+phrase "three card monte opening"
+	word
+		"You step down from your ship onto the main concourse of the spaceport, "
+		"Strolling past the smoky glow of the spaceport bar, you drift into the main area, "
+		"After a quiet walk around the area, you find yourself back in the heart of the spaceport, "
+		"You descend from your ship to enter the spaceport, "
+	word
+		"where a semicircle of onlookers has formed around a battered folding table and a trio of worn cards. "
+		"and find your path blocked by a tight ring of locals listening as a nimble-fingered dealer swaps three cards across a board. "
+		"when a sharp laugh draws your attention to a small card game on a crate. "
+		"and see a tight cluster of faces leaning in around a makeshift table as quick hands shuffle three tired cards. "
+	word
+		"The dealer looks up and grins. "
+		"The dealer glances up, eyebrows arched. "
+		"The dealer's gaze lingers on you, one hand resting on the cards as though daring you to step closer. "
+		"The dealer finally notices you, and his grin widens. "
+
+phrase "three card monte opening 2"
+	word
+		"He offers an easy challenge: "
+		"He spreads his arms over the cards in invitation: "
+		""
+		"He grandly gestures at the three waiting cards: "
+
+phrase "card anything but an ace"
+	word
+		"king"
+		"queen"
+		"jack"
+		"ten"
+		"two"
```

Comment:
Hmmm... in my mind, the King of Hearts is a proper noun and should be capitalized. I'm not sure what the standard is though, if other people disagree then this is fine as-is.

Reply:
In that case, it's probably fine in lower case.